src/block-management/ethrpc-transport-adapter.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+var nextToken = 1;
```

Comment:
Is this the same token that is attached to AbstractTransport as `nextReconnectListenerToken`?  If so, it seems confusing (to me) to track and increment them separately.  If not, I'm not clear on the purposes of the two sorts of tokens -- could you explain a bit?

Related thought: since the tokens are just integers that are incremented by one each time, could `subscriptionMapping` be replaced with an array (and nextToken/nextReconnectListenerToken simply be the length of the array)?